### ISSUE TITLE
[bug] `error-boundaries` false positives on non-react code

### ISSUE THREAD

**Issue by reteps** [Rel1cx/eslint-react#1690]
### Describe the bug

I have two questions:

1. Am I allowed to run `@eslint-react` on non-react files?

If no, you can ignore this report.

### Reproduction

```ts
import passport from 'passport';
try {
  const { getAzureStrategy } = await import('./ee/auth/azure/index.js');
  passport.use(getAzureStrategy());
} catch (err) {
 logger.error('Error initializing PrairieLearn server:', err);
 throw err;
}
```

### Expected behavior

No error

### Platform and versions

```Text
Node 24, eslint-react v4, eslint v10
```

### Stack trace

```Text
Use an Error Boundary instead of try/catch around the 'use' hook. The 'use' hook suspends the component, and its errors can only be caught by Error Boundaries.
@eslint-react/error-boundaries
```

**Comment by Rel1cx** [comment]
This is a false positive. The `use` call reported here is not in any component or hook. This issue will be fixed in the next version.
